### PR TITLE
Do not show keywords in completion after dot

### PR DESCRIPTION
**Pull request by vasily-kirichenko**

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs [hunk 1]
```diff
@@ -88,7 +88,9 @@ type internal FSharpCompletionProvider
         asyncMaybe {
             let! parseResults, parsedInput, checkFileResults = checker.ParseAndCheckDocument(filePath, textVersionHash, sourceText.ToString(), options, allowStaleResults = true)
 
+            //#if DEBUG
```
Comment (forki) [inline vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs]: ;-)

File: vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs [hunk 2]
```diff
@@ -192,7 +194,7 @@ type internal FSharpCompletionProvider
                 declarationItemsCache.Add(completionItem.DisplayText, declItem)
                 results.Add(completionItem))
 
-            if results.Count > 0 && not declarations.IsForType && not declarations.IsError then
+            if results.Count > 0 && not declarations.IsForType && not declarations.IsError && List.isEmpty qualifyingNames then
```
Comment (forki) [inline vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs]: isNil?

File: vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs (hunk 1 above)
Comment (vasily-kirichenko) [inline vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs]: Yeah... :)

File: vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs (hunk 2 above)
Comment (vasily-kirichenko) [inline vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs]: I think `isNil` is used in the compiler and FCS exclusively. We should use normal functions in FSharp.Editor.

**Comment by KevinRansom** [comment]
@vasily-kirichenko 

Thanks for this.

Kevin

**Pull request merged**